spec/models/spree/mollie/order_serializer_spec.rb

```diff
@@ -102,7 +102,7 @@
       end
 
       it 'has country' do
-        expect(subject[:country]).to eq 'US'
+        expect(subject[:country]).to match(/^.{2}\b$/)
```

Comment:
This seems like a breaking change from Spree's side? What's the format of the country in Spree 3.7? Please note that Mollie expects a certain format for the country field, or the order cannot be created: https://docs.mollie.com/guides/common-data-types#address-object

Reply:
Spree changed their country factories to generate a 2 character iso string including a digit. To counteract this the test has been adjusted. Since spree countries _can_ be freely created with an iso of the users choice we can never ensure that a valid iso is present for mollie. This part is the responsibility of the end user. We can however make sure that a 2 character iso2 is in place which we do right now.